src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -167,6 +167,7 @@ class AuthScreens extends React.Component {
 
         // Fetch some data we need on initialization
         NameValuePair.get(CONST.NVP.PRIORITY_MODE, ONYXKEYS.NVP_PRIORITY_MODE, 'default');
+        NameValuePair.get(CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER, ONYXKEYS.IS_NEW_USER, true);
```

Comment:
Hmm I think we should try to mimic the line above, where the variable name (and value) in CONST.NVP is almost identical to ONYXKEYS

Reply:
Yeah good idea